DnD-Montreal/session-tome
# Campaign Entry Observer Logic

## Description
Closes #253 
Creates a new method in the Entry Observer for creating the campaign entries
Adds logic to the created() function in the observer to handle campaign entries.

## How To Test
Open tinker
Create a campaign and attach multiple characters to it.
Create a new entry on a character via
`$someCharacter->entries()->create(['campaign_id' => $yourCampaign->id, ... , the rest of the required data])`
Observe that entries have been created on the other characters in the campaign via
`$someOtherCharacter->entries`
